Content.Server/Atmos/Components/FilterMaskComponent.cs

```diff
@@ -1,17 +1,16 @@
 ///using Content.Shared.Inventory;
-
+using Content.Server.Body.Systems;
 namespace Content.Server.Atmos.Components
```

Comment:
forgot to file scope

Reply:
not realy understand what do you mean by file scope, you want i put it in a other folder ?

Reply:
no

instead of
```cs
namespace Among
{
    public class Sus {}
}
```

```
namespace Among;

public class Sus {}
```